services/inc/services_dynalib.h

```diff
@@ -50,8 +50,8 @@ DYNALIB_FN(17, services, LED_RGB_SetChangeHandler, void(led_update_handler_fn, v
 DYNALIB_FN(18, services, log_print_direct_, void(int, void*, const char*, ...)) // Deprecated
 DYNALIB_FN(19, services, LED_GetColor, uint32_t(uint32_t, void*))
 
-DYNALIB_FN(20, services, log_message, void(int, const char*, const char*, int, const char*, void*, const char*, ...))
-DYNALIB_FN(21, services, log_message_v, void(int, const char*, const char*, int, const char*, void*, const char*, va_list))
+DYNALIB_FN(20, services, log_message, void(int, const char*, LogAttributes*, void*, const char*, ...))
+DYNALIB_FN(21, services, log_message_v, void(int, const char*, LogAttributes*, void*, const char*, va_list))
```

Comment:
I doubt that `va_list` definition may change between the compiler versions for the same target platform, as it's an ABI breaking change affecting a lot of existent libraries, including libc itself (though libc always can provide multiple variants of the same function for each ABI version if necessary).

The sole purpose of the `*_v` functions exported by the logging library is to support compile-time checking of the `Logger` printf-alike methods – not a critical functionality, but still it's nice to have it.